config.toml

```diff
@@ -44,7 +44,7 @@ disableHugoGeneratorInject = false
       unsafe = true
 
 [permalinks]
-  posts = "/posts/:year/:month/:title/"
+  posts = "/:title/"

```

Comment:
rationale?

Reply:
I found that with /posts.... the permalink did not lead to the correct place but with just /:title/, it actually led to the Dogepedia pages

Code:

`                <a href="{{.Permalink}}">
                    <button class="post-button">Read more</button>
                </a>`